PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/ui/SignInWindow.java

```diff
@@ -268,7 +256,7 @@ private void checkAccountAvailability() {
                     oauthLoginRadioButton.setEnabled(false);
                     labelOAuthLogin.setEnabled(false);
                 }
-
```

Comment:
Move downgrade

Reply:
resolved in https://github.com/microsoft/azure-tools-for-java/pull/5265